crates/ignore/src/walk.rs

```diff
@@ -591,7 +591,8 @@ impl WalkBuilder {
     ///
     /// Note that this *doesn't* return something that implements `Iterator`.
     /// Instead, the returned value must be run with a closure. e.g.,
-    /// `builder.build_parallel().run(|| |path| println!("{:?}", path))`.
+    /// `builder.build_parallel().run(||
+    ///     |path| { println!("{:?}", path); WalkState::Continue })`.
```

Comment:
Can you use `println!("{path:?}")` to shorten things? And put it on one line? Even if it exceeds my typical line length limit, I'd rather that because I've found newlines inside of inline code fences to sometimes produce weird rendering behavior.

Reply:
Applied